components/SwipeRow.js

```diff
@@ -177,7 +179,7 @@ class SwipeRow extends Component {
 			}
 		).start( _ => {
 			if (toValue === 0) {
-				this.props.onRowDidClose && this.props.onRowDidClose();
+				this.props.onRowDidClose && this.props.onRowDidClose(this._releaseValue);
```

Comment:
Hm am I missing something? I see that you changed `onRowOpen` and `onRowDidOpen` to accept a new arg but you're only passing the value to `onRowDidClose`?